analyzers/tests/SonarAnalyzer.UnitTest/TestCases/BlazorQueryParameterRoutableComponent_Partial.razor

```diff
@@ -0,0 +1 @@
+﻿@page "/component"
```

Comment:
I suggest for all the .razor files adding the namespace explicitly:
`@namespace EmptyProject`